src/main/java/com/mongodb/kafka/connect/source/MongoSourceConfig.java

```diff
@@ -194,6 +196,17 @@ public class MongoSourceConfig extends AbstractConfig {
           + "stream document. Automatically, sets `change.stream.full.document=updateLookup` so updated documents will be included.";
   private static final boolean PUBLISH_FULL_DOCUMENT_ONLY_DEFAULT = false;
 
+  public static final String FULL_DOCUMENT_BEFORE_CHANGE_CONFIG =
+      "change.stream.full.document.before.change";
+  private static final String FULL_DOCUMENT_BEFORE_CHANGE_DISPLAY =
+      "The fullDocumentBeforeChange configuration.";
+  private static final String FULL_DOCUMENT_BEFORE_CHANGE_DOC =
+      "Specifies the pre-image configuration when creating a Change Stream.\n"
+          + "The pre-image is not available in source records published while copying existing data as a result of"
+          + " enabling `copy.existing`, and the pre-image configuration has no effect on copying.\n"
+          + "See https://www.mongodb.com/docs/manual/reference/method/db.collection.watch/ for more details and possible values.";
```

Comment:
I linked the server docs here instead of listing and explaining all possible values, because, as always, duplicating the docs eventually leads to the duplicated docs becoming inconsistent and/or obsolete, which results in misleading users, which in turn, makes users to distrust the docs.

In the past, I gave examples of problems our approach introduced in the driver API (where the decision has been and stays "the convenience of reading docs in place is more important than having consistent docs users may trust"), and we have similar situations in Kafka. For example, the documentation for `change.stream.full.document` config property states that possible values are only `""`, `"default"`, `"updateLookup"`. The new values `"whenAvailable"` and `"required"` are omitted, as they were added later - what should users thing about this? Does the connector not support these values? Does it support them, but the authors forgot to specify them? If users actually trust the connector docs (they should not), they will not learn about these two new values, or any values that will be supported by the server in the future.

Reply:
Ack